thunder/torch/__init__.py

```diff
@@ -1031,6 +1031,11 @@ def view(a: TensorLike, /, *shape) -> TensorLike:
     return reshape(a, shape)
 
 
+@torchsymbol(torch.Tensor.view_as, is_method=True)
+def view_as(a: TensorLike, /, b: TensorLike) -> TensorLike:
```

Comment:
nit: `a: TensorLike, b: TensorLike, /)` — the reason for this is protect us from people writing code like `view_as(b=...)` and having to support the parameter name "b" forever, even though the name doesn't really matter in this case

Reply:
Oh my bad. I'll make sure to check on this next time! Thanks!